client/src/views/Home/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import MovieItem from "./MovieItems";
+import SearchBar from "../../components/SearchBar";
+import HomeText from "../../components/HomeText";
+import SearchError from "../../components/SearchError";
+import Loader from "../../components/Loader";
+import Illustration from "../../images/2.Illustrations/illustration-empty-state.png";
+import "./styles.scss";
+import axios from "axios";
+
+const Home = ({ favorites }) => {
+  const [searchText, setSearchText] = useState("");
+  const [moviesList, setMoviesList] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [searchError, setSearchError] = useState(false);
+
+  useEffect(() => {
+    if (searchText) {
+      const httpRequest = setTimeout(async () => {
```

Comment:
The purpose of this `setTimeout` is to create a denounce, isn't it? If so, it's a good way to avoid excessive requests.

Reply:
Hi Danilo! Yes, that was exactly my goal, to make the request only some time the user has finished typing